CoreFoundation/Base.subproj/CoreFoundation.h

```diff
@@ -33,6 +35,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <netdb.h>
```

Comment:
I’ve added netdb.h in the copy of this header in the SwiftRuntime directory with a #if __has_include(..)